documentation/book/assembly-using-kafka-connect-with-plugins.adoc

```diff
@@ -9,10 +9,11 @@
 [id='using-kafka-connect-with-plug-ins-{context}']
 = Extending Kafka Connect with connector plug-ins
 
-The {ProductName} container images for Kafka Connect include the two built-in file connectors: `FileStreamSourceConnector` and `FileStreamSinkConnector`. You can add your own connectors by using one of the following methods:
+The {ProductName} container images for Kafka Connect include the two built-in file connectors: `FileStreamSourceConnector` and `FileStreamSinkConnector`.
+You can add your own connectors by:
 
-* Create a Docker image from the Kafka Connect base image.
-* Create a container image using {OpenShiftName} builds and Source-to-Image (S2I).
+* Creating a Docker image from the Kafka Connect base image.
```

Comment:
I don't think this adds much. We'd need to define the term CI, and it doesn't matter from our point of view how the container image is created. Should we be using the term "container image" rather than "Docker image" everywhere?

Reply:
_container image_ is vendor neutral and might make it more clear that it is not exclusively Docker. We can leave it as it is if you want. But I felt like it needed some more explanation what the user will be doing.

Reply:
We know that most of the people talk about Docker container but I totally agree on changing it to container image everywhere.

Reply:
```suggestion
* Creating a container image from the Kafka Connect base image (manually or using your CI (continuous integration), for example).
```